src/common/process_task_results/get_results/script.R

```diff
@@ -61,9 +61,12 @@ trace <- readr::read_tsv(par$input_execution) %>%
     dataset_id = stringr::str_extract(id, id_regex, 2L),
     normalization_id = stringr::str_extract(id, id_regex, 3L),
     method_id = stringr::str_extract(id, id_regex, 4L),
-  ) %>%
-  filter(process_id == method_id)
-
+    submit = strptime(submit, "%Y-%m-%d %H:%M:%S"),
+  )  %>%
+  filter(process_id == method_id) %>%
+  arrange(submit) %>%
+  group_by(name) %>%
+  slice(n())
```

Comment:
Looks very good! I have a minor cosmetic suggestion:

```suggestion
  ) %>%
  filter(process_id == method_id) %>%
  arrange(desc(submit)) %>%
  group_by(name) %>%
  slice(1)
```